lsp-headerline.el

```diff
@@ -173,9 +173,15 @@ Switch to current mouse interacting window before doing BODY."
                                      (lsp-headerline--go-to-symbol symbol)))
                                  (define-key map [header-line mouse-2]
                                    (lsp-headerline--make-mouse-handler
-                                     (lsp-headerline--narrow-to-symbol symbol)))
+                                     (if (buffer-narrowed-p)
```

Comment:
Doesn't it make sense to widen only if you are trying to narrow to the exact same region? In case you want to narrow to another item(e. g. parent), it seems like it makes more sense to narrow to that region.

Reply:
Oh, You are right. I've never seen multiple (parent and child) symbols in headerline. 
I try to fix my code.